tensornetwork/network_components.py

```diff
@@ -569,61 +569,43 @@ def op_protection(self, other: Union[int, float, "Node"]) -> "Node":
                             self.backend.name, other.backend.name))
       if not hasattr(other, '_tensor'):
         raise AttributeError("Please provide a valid tensor for this Node.")
-    else:
-      other_tensor = self.backend.convert_to_tensor(other)
-      other = Node(tensor=other_tensor, backend=self.backend.name)
+      return other._tensor
     return other
 
   def __add__(self, other: Union[int, float, "Node"]) -> "Node":
-    other = self.op_protection(other)
-    new_tensor = self.backend.addition(self.tensor, other.tensor)
-    if len(self.axis_names) > len(other.axis_names):
-      axis_names = self.axis_names
-    else:
-      axis_names = other.axis_names
+    other_tensor = self.op_protection(other)
+    new_tensor = self.backend.addition(self.tensor, other_tensor)
     return Node(
         tensor=new_tensor,
         name=self.name,
-        axis_names=axis_names,
+        axis_names=None,
```

Comment:
Is there a reason you changed this?

Reply:
I feel using the axis names of self is somewhat arbitrary (why not the ones of other?), and can confuse users. This way seems more natural.